lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -2965,6 +2965,7 @@ void DWARFASTParserClang::ParseSingleMember(
   // data members is DW_AT_declaration, so we check it instead.
   // FIXME: Since DWARFv5, static data members are marked DW_AT_variable so we
   // can consistently detect them on both GCC and Clang without below heuristic.
+  // Remove this block if we ever drop DWARFv4 support.
```

Comment:
FWIW this seems extremely unlikely. It doesn't really matter, but I would've just said that the next block is specific to DWARF4.

Reply:
Clarified here: https://github.com/llvm/llvm-project/pull/72495